mitosheet/src/components/taskpanes/Graph/GraphSidebar.tsx

```diff
@@ -254,26 +243,42 @@ const GraphSidebar = (props: {
         changeLoadingGraph(getGraphAsync);
     }
 
+    /*
+        Updates the graph params on undo so that the graph configuration is in sync
+        with the graph shown
+    */
+    const refreshParamsAfterUndo = async (): Promise<void> => {        
+
+        // We also set ignoreNextUpdate to true, so that these updates
+        // don't cause a new message to get sent, as then we get trapped
+        // in an infinite loop where an undo will cause a new graph message
+        // to get sent 
```

Comment:
Why would this cause an infinite loop? Shouldn't this only be called when there was an undo?